### PR TITLE
Include instances not attached to an ASG

### PR DESCRIPTION
**Pull request by lindseydew**
So that we can monitor the costs of our EMR clusters

cc @philwills @obrienm

### CODE REVIEW

File: app/lib/FutureO.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package lib
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+case class FutureO[T](futureOption: Future[Option[T]]) {
```
Comment (philwills) [inline app/lib/FutureO.scala]: YolO

**Comment by philwills** [comment]
As @tackley would say, 👍 once it compiles.

**Comment by tackley** [comment]
👍 once it compiles

**Comment by lindseydew** [comment]
hmm the build is still failing on Shippable due to an unresolved dependency. 
It is also failing in master, but the console output is blank.
https://app.shippable.com/runs/56fa826f92cf2e0d004d94b4

Maybe we could stop using Shippable?

**Comment by philwills** [comment]
Yea, sorry, that one was an experiment, which I should have removed.

**Comment by lindseydew** [comment]
Ace thank you!

**Pull request merged**